### PR TITLE
detekt 1.3.0 -> 1.5.1

### PR DESCRIPTION
**Pull request by wasanx25**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/detekt.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Detekt < Formula
   desc "Static code analysis for Kotlin"
   homepage "https://github.com/arturbosch/detekt"
-  url "https://jcenter.bintray.com/io/gitlab/arturbosch/detekt/detekt-cli/1.3.0/detekt-cli-1.3.0-all.jar"
-  sha256 "610b0211b909d8e79c1b623119e1d94cb833a8e98b32e5a0e991fa2d214992f6"
+  url "https://jcenter.bintray.com/io/gitlab/arturbosch/detekt/detekt-cli/1.5.1/detekt-cli-1.5.1-all.jar"
+  sha256 "7bfff3b000cc798f44355ef29499f05625f72250c667c56aa3d571b0bb9567d7"
   revision 1
```
Comment (SMillerDev) [inline Formula/detekt.rb]: This should be removed

**Review by SMillerDev — COMMENTED** [review]
Almost there

File: Formula/detekt.rb (hunk 1 above)
Comment (wasanx25) [reply Formula/detekt.rb]: I fixed

**Comment by SMillerDev** [comment]
Could you squash your commits please?

**Comment by zbeekman** [comment]
> Could you squash your commits please?

I'll do it during merge

**Comment by zbeekman** [comment]
Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you @wasanx25!

**Comment by zbeekman** [comment]
@wasanx25: In the future, please:

1. Squash all your commits into one and force push to your branch (or just use `git commit --amend` when making changes)
2. Use the standard git commit message: `detekt 1.5.1`

In general you don't need to add more info in the commit message, but if you do, make sure the first line matches the format `detekt <version>` and then leave a blank line and add further details on the third (3rd) line or after.

**Comment by wasanx25** [comment]
@zbeekman Sorry, I didn't read the CONTRIBUTING.md carefully. I understand the rule, I will be more careful.

Thanks,

**Pull request closed without merging**